client/jetpack-onboarding/steps/woocommerce.jsx

```diff
@@ -27,6 +27,12 @@ class JetpackOnboardingWoocommerceStep extends React.PureComponent {
 		this.props.saveJetpackOnboardingSettings( this.props.siteId, {
 			installWooCommerce: true,
 		} );
+
+		this.props.recordJpoEvent( 'calypso_jpo_woocommerce_install_clicked' );
+	};
+
+	handleWooCommerceInstallationSkip = () => {
+		this.props.recordJpoEvent( 'calypso_jpo_woocommerce_dont_install_clicked' );
```

Comment:
Maybe `_skip_` instead of `_dont_`? Not sure.

Reply:
I was hesitant because we have a different skip button, but it has a different event name, so it should be good. Addressed in https://github.com/Automattic/wp-calypso/pull/22389/commits/d8fc776045e7819f18e01af1805c47f053cb5c58.